docs/new/references/flashblocks.md

```diff
@@ -0,0 +1,148 @@
+
+If the user requested `include_partial_blocks` (and NOT `partial_blocks_only`), they will also receive:
+
+5. The result of execution of trx 0-100, within BlockData with Clock (num=123, ID=0xdddddddddd) <- Note that the ID here is the same
+
+
+## Using Flashblocks
+
+### A simple test, from terminal, with `substreams run`
+
+1. Get the latest release of Substreams: https://github.com/streamingfast/substreams/releases/tag/v1.17.8
+
+2. To test with a common module, using `jq` to quickly see what is going on (you need [jq](https://jqlang.org/)):
+
+`substreams run -e https://base-mainnet-flash.streamingfast.io ethereum_common all_events -s -1 --include-partial-blocks -o jsonl | jq -r '"Block: #\(.["@block"]) Partial: \(.["@partial_index"]) Event count:\(.["@data"].events|length)"'`
+
+This will print lines like this:
+
+```
+Block: #39306107 Partial: 5 Event count:531
+Block: #39306107 Partial: 7 Event count:152
+Block: #39306107 Partial: 9 Event count:182
+Block: #39306107 Partial: 10 Event count:127
+Block: #39306107 Partial: null Event count:992
+Block: #39306108 Partial: 3 Event count:899
+Block: #39306108 Partial: 5 Event count:197
+Block: #39306108 Partial: 7 Event count:107
+Block: #39306108 Partial: 9 Event count:121
+Block: #39306108 Partial: 10 Event count:77
+Block: #39306108 Partial: null Event count:1401
+```
+
+When you see "partial: null" it means that it is the actual full block.
+
+3. To see how it performs with a clock, you can use, as always, the -o clock with something like this:
+
+`substreams run -e https://base-mainnet-flash.streamingfast.io https://github.com/graphprotocol/graph-node/raw/refs/heads/master/substreams/substreams-head-tracker/substreams-head-tracker-v1.0.0.spkg -s -1 -o clock --include-partial-blocks`
+
+This will print lines like this:
+```
+----------- PARTIAL BLOCK #39,306,388 (idx=7) (bcbcc30d0d6f4d5581519f0da90fe8b61ab8ae4e38193c1714285bee2be5cb98) age=-484.276ms ---------------
+----------- PARTIAL BLOCK #39,306,388 (idx=9) (0399532628e8b7585c53a58b3bce5d6432fc76b3aee4730292ab4cef8c675ccf) age=-181.9ms ---------------
+----------- PARTIAL BLOCK #39,306,388 (idx=10) (c7871f81180ad6565ae6f9c4d244b2dcac05b2348aff54bd591a65d6c945107e) age=1.138322s ---------------
+----------- BLOCK #39,306,388 (c7871f81180ad6565ae6f9c4d244b2dcac05b2348aff54bd591a65d6c945107e) age=1.138495s ---------------
+```
+See the "negative age", that's because at partial block with idx=5, the proposed block timestamp is still 2 seconds in the future.
+
+### A more useful example, with the Substreams Webhook Sink:
+
+1. Get the latest release of Substreams: https://github.com/streamingfast/substreams/releases/tag/v1.17.8
+2. Run this command:
+
+`substreams sink webhook --partial-blocks-only -e https://base-mainnet-flash.streamingfast.io http://webhook.example.com path-to-your.spkg  -s -1`
+
+Enjoy!
+
+### More sinks
```

Comment:
I think it would be good to "expand" that here and show it inline. It would also remove most of the `### More sinks` section actually.

It would probably be good to also update the Substreams JS SDK for this.